source/docs/software/vscode-overview/3rd-party-libraries.rst

```diff
@@ -103,6 +103,22 @@ The WPILib :doc:`command library </docs/software/commandbased/index>` has been s
       * PyPI package: ``robotpy[commands2]`` or ``robotpy-commands-v2``
       * In ``pyproject.toml``: ``robotpy_extras = ["commands2"]``
 
+[ChoreoLib](https://sleipnirgroup.github.io/Choreo/choreolib/installation/) - Library for reading and following trajectories generated by [Choreo](https://sleipnirgroup.github.io/Choreo/)
+
+ChoreoLib offers support for creating autonomous paths using [Choreo](https://sleipnirgroup.github.io/Choreo/).
```

Comment:
ChoreoLib doesn't create paths though. It reads trajectories generated by Choreo.

Reply:
Yeah I should have said creating autonomous routines I'll change it